### DISCUSSION TITLE
posts/coreprotect-mc-1-21-3/

### DISCUSSION THREAD

**Discussion by giscus** [DarrenOfficial/DarrenOfficial#15] [General]
# posts/coreprotect-mc-1-21-3/

CoreProtect for 1.21.x release - jar file

https://blog.darrennathanael.com/posts/coreprotect-mc-1-21-3/

**Comment by sudofox** [comment]
Thank you for the help, wonder why CoreProtect hasn't released the latest version yet

**Reply by littleosman** [reply]
paywall is the answer. greed arrived at minecraft

**Comment by DarrenOfficial** [comment]
They have released it, only on Patreon

**Comment by RainbowwwJ** [comment]
goat
